index.Rmd

```diff
@@ -1,105 +1,228 @@
+  theme(axis.text = element_blank(),
+        axis.ticks = element_blank(),
+        panel.background = element_blank(),
+        panel.grid.minor = element_blank()) +
+  labs(title = "Water or Water ice or Volatiles materials", 
+       x = " ", 
+       y = " ") +
+  scale_fill_gradient(high = "#CEE50E", 
+                      low = "#087F28",
+                      name = "Water or Water ice or Volatiles materials")
+```
+
+## PART2: Visualization of mining sites within the selected landing site on Martian gloge (map) 
+
+After knowing that Lyot crater"V00823007" site is the best site to be consider as best future landing site according to the figure above, We just get the GPS coordinates of some locations within the Lyot crater"V00823007" site  (GPS coordinates from the onboard GPS of the spacecraft), created an exel file containing the coordinates of those locations and plot them on Martian georeferenced globe using the threejs package.
+As state previously, the locations within Lyot crater"V00823007" shown on the Martian gloge are the future mining site because volatile materials are more abundants in those locations compare to other location within Lyot crater"V00823007".
+
+I loaded required packages
+
+```{r, message=F, warning=F}
+library(threejs)
 ```
 
+##Results2
+I firstly prepared and loaded the data and then with the package called "threejs" I plot the exact locations of  mining sites on the Martian global map. I created a cvs file where I stored the latitudes and logitude of the future  mining sites within Lyot crater"V00823007" site and then call that file in R.
+
 
 ```{r}
-data %>% 
-  ggplot(aes(x=x,y=y,col=category))+
-  geom_point()
+library(threejs)
+library(csvread)
+the_martian <- read.csv("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landingMining_sites/the_martian.csv", comment.char="#")
+                        
+#the_martian.csv <- ("C:/Users/Owner/Desktop/sare  backup/c/user #hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landi#ngMining_sites/the_martian.cvs")
+  
+
+globejs(img = "C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landingMining_sites/Mars_Viking_MDIM21_ClrMosaic_global_1024.jpg", bodycolor="#555555", emissive="#444444",
+        lightcolor="#555555",
+        lat = the_martian$x,
+        long = the_martian$y)
+
 ```
 
+
+
 # Conclusions
+The selection of the best landing site is firstly done by choosing training sites and processing satellites images to see where the water or volatiles materials are abundant. Secondly, we  get the geographical coordinates of the locations consider to be best mining sites within Lyot crater"V00823007" and plot them on a Martian globe using the package threej. 
```

Comment:
But this was not done in this script!  You loaded and plotted some images (and a related index) and then loaded and plotted some points.  There was no analysis.